src/test/java/definitions/ChengyuRegistrationFirstNameStepdefs.java

```diff
@@ -0,0 +1,77 @@
+package definitions;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.openqa.selenium.By;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static support.TestContext.getDriver;
+
+public class ChengyuRegistrationFirstNameStepdefs {
+    @Given("I open {string} page")
+    public void iOpenPage(String url) {
+        if(url.equals("registration")){
+            getDriver().get("http://ask-qa.portnov.com/#/registration");
+        }else if (url.equals("login")){
+            getDriver().get("http://ask-qa.portnov.com/#/login");
+        }else {
+            System.out.println("This site is not supported: " + url);
+        }
+    }
+
+    @When("I type {string} into first name field")
+    public void iTypeIntoFirstNameField(String firstName) {
+        getDriver().findElement(By.xpath("//input[@formcontrolname='firstName']")).sendKeys(firstName);
+    }
+
+    @Then("I click the {string} button")
+    public void iClickTheButton(String btnName) {
+        getDriver().findElement(By.xpath("//span[contains(text(),'"+btnName+"')]")).click();
+    }
+
+
+    @Then("error message {string} is displayed")
```

Comment:
One more duplicated  method on line 35  public void errorMessageIsDisplayed(String expectedMessage) 
which is already present in quizMultipleChoiceOlga.  Please delete or comment out this method.  
Every time you make a new push to origin you have to switch to you local master, do git pull, then go back to your branch, do git merge master and check if any conflicts or duplication of methods occur.
Pls. update your branch, create a new commit and  a new PR.